src/data/db.ts

```diff
@@ -0,0 +1,225 @@
+//       data: {
+//         questionId: 'id71',
+//         text: 'Lorem <suscipti> dolor sit amet, consectetur adipiscing elit. <Praesent> ut fermentum nulla, sed <ip sum> sem.',
+//       },
+//     }),
+//     mockAppDataFactory({
+//       id: '10',
+//       item: mockItem,
+//       member: mockMembers[4],
+//       creator: mockMembers[4],
+//       data: {
+//         questionId: 'id61',
+//         text: 'bird',
+//       },
+//     }),
+//   ],
+//   appSettings: [
+//     mockAppSettingFactory({
+//       id: 'question-list',
+//       item: mockItem,
+//       data: {
+//         list: ['id71', 'id51', 'id61', 'id41'],
+//       },
+//       name: APP_SETTING_NAMES.QUESTION_LIST,
+//     }),
+//     mockAppSettingFactory({
+//       id: 'id4',
+//       item: mockItem,
+//       data: {
+//         questionId: 'id41',
+//         question: 'What is the capital of France?',
+//         type: QuestionType.MULTIPLE_CHOICES,
+//         choices: [
+//           {
+//             value: 'London',
+//             isCorrect: false,
+//             explanation: 'London is the capital of England',
+//           },
+//           {
+//             value: 'Paris',
+//             isCorrect: true,
+//             explanation: 'Paris is the capital of France',
+//           },
+//           {
+//             value: 'New York',
+//             isCorrect: false,
+//             explanation: 'New York is not located in France',
+//           },
+//           {
+//             value: 'Tokyo',
+//             isCorrect: false,
+//             explanation: 'Tokyo is the capital of Japan',
+//           },
+//         ],
+//         explanation: 'Paris is the capital of France.',
+//       },
+//       name: APP_SETTING_NAMES.QUESTION,
+//     }),
+//     mockAppSettingFactory({
+//       id: 'id5',
+//       item: mockItem,
+//       data: {
+//         questionId: 'id51',
+//         question: 'How happy are you?',
+//         type: QuestionType.SLIDER,
+//         min: 10,
+//         max: 90,
+//         value: 20,
+//         explanation: 'Go to sleep.',
+//       },
+//       name: APP_SETTING_NAMES.QUESTION,
+//     }),
+//     mockAppSettingFactory({
+//       id: 'id6',
+//       item: mockItem,
+//       data: {
+//         questionId: 'id61',
+//         question: 'What is a baby cat called?',
+//         type: QuestionType.TEXT_INPUT,
+//         text: 'kitten',
+//       },
+//       name: APP_SETTING_NAMES.QUESTION,
+//     }),
+//     mockAppSettingFactory({
+//       id: 'id7',
+//       item: mockItem,
+//       data: {
+//         questionId: 'id71',
+//         question: 'Fill In The Blanks',
+
+//         type: QuestionType.FILL_BLANKS,
+//         text: 'Lorem <ips um> dolor sit amet, consectetur adipiscing elit. <Praesent> ut fermentum nulla, sed <suscipit> sem.',
+//       },
+//       name: APP_SETTING_NAMES.QUESTION,
+//     }),
+//   ],
+//   members: mockMembers,
+//   appActions: [],
+//   items: [mockItem],
```

Comment:
Remove this file?

Reply:
Oh yes I forgot this... I will remove it.